guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -30,6 +30,62 @@ The release notes include:
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
 
+## v2002.0.16
+
+-   {:.new}**Docker updates**—
+
+    -   <!-- MAGECLOUD-3025 -->Now, the default service configuration generated in the Docker environment is the same as the default configuration in the Cloud template.
+
+    -   <!-- MAGECLOUD-2907 -->You can send mail from your Docker environment using the [`sendmail` service]({{page.baseurl}}/cloud/docker/docker-development.html#sendmail-service).
+
+    -   <!-- MAGECLOUD-2891 -->Added the ability to [configure Xdebug]({{page.baseurl}}/cloud/docker/docker-development-debug.html) to debug in the Cloud Docker environment.
+
+    -   <!-- MAGECLOUD-2883 -->Fixed an issue with web service permissions when generating the `docker-compose.yml` file.
+
+-   {:.new}<!-- MAGECLOUD-2392 -->**Upgrade improvement**—Added validation to confirm that the `autoload` property in the `composer.json` file contains required configuration changes before upgrading to {{ site.data.var.ee }} v2.3. See [Upgrade Magento version]({{site.baseurl }}/guides/v2.3/cloud/project/project-upgrade.html).
+
+-   {:.fix}<!-- MAGECLOUD-3104 -->Fixed a compression issue when generating assets during builds by moving the compression task to the beginning of the `build:transfer` section of the build phase.
```

Comment:
There was no bug here. We just made the allocation of processes to sub-phases more logical. This change made possible the new use case, which was required for custom bundling/minification processes.

Now you can minify/bundle your static assets, while native process to compress those comes in transfer phase.

Being short, this change enables use case described in https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_magento-app.html#hooks (customized build process section)